src/PaymentBundle/Resources/docs/reference/configuration.rst

```diff
@@ -117,7 +117,7 @@ Full Configuration Options
         selector: sonata.payment.selector.simple
 
         # service which generate the correct order and invoice number
-        generator: sonata.payment.generator.mysql
+        generator: sonata.payment.generator.mysql # or sonata.payment.generator.postgres
```

Comment:
>  So any other service can be passed there.

And it should, right? Or are you suggesting restricting the list to only our implementations, @OskarStark ?